.test-infra/jenkins/job_PerformanceTests_KafkaIO_IT.groovy

```diff
@@ -61,14 +61,38 @@ job(jobName) {
     autoscalingAlgorithm         : 'NONE'
   ]
 
+  Map runnerV2SdfWrapperPipelineOptions = pipelineOptions + [
+    kafkaTopic                   : 'beam-runnerv2',
+    bigQueryTable                : 'kafkaioit_results_sdf_wrapper',
+    influxMeasurement            : 'kafkaioit_results_sdf_wrapper',
+    experiments                  : 'beam_fn_api,use_runner_v2,use_unified_worker',
+  ]
+
+  Map runnerV2SdfPipelineOptions = pipelineOptions + [
+    kafkaTopic                   : 'beam-sdf',
+    bigQueryTable                : 'kafkaioit_results_runner_v2',
+    influxMeasurement            : 'kafkaioit_results_runner_v2',
+    experiments                  : 'beam_fn_api,use_runner_v2,use_unified_worker,use_sdf_kafka_read',
+  ]
+
   steps {
     gradle {
       rootBuildScriptDir(common.checkoutDir)
       common.setGradleSwitches(delegate)
       switches("--info")
-      switches("-DintegrationTestPipelineOptions=\'${common.joinOptionsWithNestedJsonValues(pipelineOptions)}\'")
+      switches("-DintegrationTestPipelineOptions=\'${common.joinOptionsWithNestedJsonValues(runnerV2SdfWrapperPipelineOptions)}\'")
+      switches("-DintegrationTestRunner=dataflow")
+      switches("-Dexperiment=use_runner_v2")
```

Comment:
What actually "runner v2" is?

Reply:
`runner v2` is a new version of Dataflow. We are under migration from old Dataflow runner to this new runner. We(dataflow) are more interested in making sure SDF and KafkaIO work well on the new runner.

Reply:
Hmm, maybe it's not related to this PR, but the value `use_runner_v2` doesn't reflect that it's related only to Dataflow, I'd assume mostly Runner v2 API looking on this. Do you know if it's already defined term?

Reply:
That's a good point. The Dataflow backend expects that exact string but for Beam users `dataflow_v2` makes more sense. It is just an experiment and temporary, though. Hopefully most users will never need to set it. We could add the additional string and just swap it out when we submit the job.

Reply:
Imho, it's fine if it's just temporary but if it's going to be exposed to user or even for the sake of clarity, I'd prefer to have additional string value, like `dataflow_v2` for instance.

Reply:
For testing, we need this string to decided whether to build the sdk container. In the latest change, I have removed this argument and checked pipeline options string only.